utilities/claim/claim.js

```diff
@@ -187,6 +188,8 @@ async function initiateRound(privateKey, { ethRegistryAddress, ethTokenAddress,
       }
       const isComplete = await tokenBridge.isTransferCompleted(vaa)
       console.log(`Wormhole redemption completion status: ${isComplete}`)
+      // Wait for 30 seconds to ensure the VAA is processed
+      await new Promise((resolve) => setTimeout(resolve, 30_000))
```

Comment:
Huh. This is interesting... We get back a vaa but it isn't processed?

Reply:
yeah.... i kinda think it maybe is related to some rpc drift? will take a closer look upon moving the plugin but this same code failed and then worked a few sec later